lib/rules/multiline-comment-style.js

```diff
@@ -272,6 +272,7 @@ module.exports = {
                         if (
                             comment.type === "Line" &&
                             index && commentList[index - 1].type === "Line" &&
+                            sourceCode.getTokenBefore(comment, { includeComments: true }).loc.end.line === comment.loc.start.line - 1 &&
```

Comment:
Suggestion: Maybe assign `sourceCode.getTokenBefore(comment, { includeComments: true })` to a variable that can be reused since this token is used for two checks in this conditional?

Reply:
sounds good!

Reply:
Also, this won't ever be `undefined` because of [this line](https://github.com/eslint/eslint/pull/9475/files#diff-e8cce2004dd5e0386021b0e80370c8ebR269), correct?

Reply:
hmm.. I'm afraid I don't think so -- `tokenBefore` being null, returning `!tokenBefore` can be true.

Reply:
Ah, right - just to be sure, can we either do:

```js
const tokenBefore = sourceCode.getTokenBefore(comment, { includeComments: true });

if (
  ...
  tokenBefore && tokenBefore.loc.end.line === comment.loc.start.line - 1 &&
  ...
)
```
Or right a test that tests that ensure this doesn't throw? It might not ever be `null` or `undefined`, in which case we're fine!

Reply:
>but prev line can make sure: index && ... index>0 means at least one comment.
all right. it's a little tricky, no harm to add this check.